### ISSUE TITLE
JSR doesn't support types

### ISSUE THREAD

**Issue by AkioSarkiz** [Hexagon/croner#258]
**Describe the bug**
If you install the package via JSR registry you can see that types are not included

**To Reproduce**
- Create a bun project
- Install the package via JSR registry

```
Could not find a declaration file for module '@hexagon/croner'. '/Volumes/source_code/project/node_modules/@hexagon/croner/dist/croner.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/jsr__hexagon__croner` if it exists or add a new declaration (.d.ts) file containing `declare module '@hexagon/croner';`
```

**Expected behavior**
There are two ways, first is include types into the JSR package, second one is create a new package with types

**Comment by Hexagon** [comment]
Interesting, guess i'll have to move the types from `/types` to `/dist` to have the types adjacent to the js, as there is no package.json pointing to the types.

**Comment by Hexagon** [comment]
I'll convert the code base to TypeScript-first, which will eliminate slow types and indirectly resolve this.

**Comment by Hexagon** [comment]
Fixed in `@hexagon/croner` through `9.0.0` at jsr.io
